projects/media-viewer/src/lib/annotations/comment-set/comment-set-header/comment-set-header.component.ts

```diff
@@ -0,0 +1,18 @@
+import {Component, EventEmitter, Input, Output, ViewEncapsulation} from '@angular/core';
+
+@Component({
+  selector: 'mv-comment-set-header',
+  templateUrl: './comment-set-header.component.html',
+  styleUrls: ['./comment-set-header.component.scss'],
+  encapsulation: ViewEncapsulation.None
```

Comment:
is this needed?

Reply:
yes it is otherwise the css will not get picked up...